### DISCUSSION TITLE
Let the user exclude search results having specific patterns by using a - (minus) prefix, as in Google results

### DISCUSSION THREAD

**Discussion by raindropsfromsky** [Flow-Launcher/Flow.Launcher#1188] [Ideas]
We should be able to exclude any pattern by using a hyphen (minus) prefix, as in Google search. This will reduce the number of hits, and thus reduce the clutter.

**Comment by taooceros** [comment]
Interesting idea. Do you want plugin specific or simply global?

**Comment by raindropsfromsky** [comment]
Global, please!

**Reason:** The terms we want to exclude would not change based on the choice of our search domain.
And different plugins in FL merely search in different domains.
Therefore the search string would be composed identically for all search-related plugins.

**Comment by raindropsfromsky** [comment]
BTW if we want to retain the hyphen as a searchable character, the "exclude" operator can be changed to NOT or \~ (or any other).
